Lesson Modules/Module 6- Data Binning.ipynb

```diff
@@ -101,7 +101,7 @@
     "    binnedCPM = np.array(sumCPM) / np.array(DataCount)          # np.array allows us to perform element-by-elemtent division\n",
     "    \n",
     "    avgError = np.array(sumError) / np.array(DataCount)\n",
-    "    stdError = avgError / np.sqrt(len(DataCount))               # standard error is average error divided by sqrt(# of elements)\n",
+    "    stdError = avgError / np.sqrt(DataCount)               # standard error is average error divided by sqrt(# of elements)\n",
```

Comment:
This looks right although it looks like the plot hasn't updated yet?

This is the updated plot I get on my laptop:
![screenshot from 2016-08-01 18 31 33](https://cloud.githubusercontent.com/assets/5259089/17314066/43ad1d64-5816-11e6-9a6d-2756f05da215.png)